### ISSUE TITLE
Add realtime feedback when searching for games

### ISSUE THREAD

**Issue by Brett208** [OutpostUniverse/NetFixClient#6]
Currently, when searching for a game, some text appears saying searching...

There is no indication that the program hasn't hung up. I think it would improve the user experience to give a visual update.

Perhaps
- Searching
- Searching.
- Searching..
- Searching...
- Searching
- and so on

Or maybe a timer in seconds
 - Searching (1)
 - Searching (2)
 - and so on

I haven't looked into feasibility of this with code base, but would be useful based on my own experiences with the application.

**Comment by DanRStevens** [comment]
I agree. The lack of sufficient feedback there has long been an issue.
